app/flash/FlashMessage.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
```

Comment:
What's the reason behind having this component in it's own folder, as opposed to being in the `components` folder?